### DISCUSSION TITLE
Welcome to video-downloader-deploy Discussions!

### DISCUSSION THREAD

**Discussion by LussacZheng** [LussacZheng/video-downloader-deploy#19] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * ❓ Ask questions you’re wondering about.
  * 💡 Share ideas.
  * 💬 Engage with other community members.
  * 💪 Welcome others and are open-minded. Remember that this is a community we build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community, or open a [new discussion](https://github.com/LussacZheng/video-downloader-deploy/discussions/new).

**Comment by MYNVFYCR** [comment]
此应用无法在你的电脑上运行
若要找到适用于你的电脑的版本，请咨询软件发布者。
Windows 10 企业版 LTSC       版本号  21H2

**Reply by LussacZheng** [reply]
@MYNVFYCR 网络问题，下载 lux 的时候没有正常完成。可以从项目主页的蓝奏网盘里下载完整版（带有`[FULL]`的）来重新进行部署。

**Comment by MYNVFYCR** [comment]
网络有问题&nbsp; 所有的蓝奏云链接都打不开&nbsp; &nbsp;换了几个网也不行&nbsp; 不知道怎么回事


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "LussacZheng/video-downloader-deploy"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月7日(星期六) 凌晨0:45
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [LussacZheng/video-downloader-deploy] Welcome to video-downloader-deploy Discussions! (Discussion #19)





 
@MYNVFYCR 网络问题，下载 lux 的时候没有正常完成。可以从项目主页的蓝奏网盘里下载完整版（带有[FULL]的）来重新进行部署。

**Reply by LussacZheng** [reply]
换浏览器试试

**Comment by MYNVFYCR** [comment]
油猴插件问题&nbsp; 禁用掉就好了 谢谢回复




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年5月7日(星期六) 上午8:56
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [LussacZheng/video-downloader-deploy] Welcome to video-downloader-deploy Discussions! (Discussion #19)





 
换浏览器试试
